sdk/webpubsub/Microsoft.Azure.WebPubSub.Common/src/Microsoft.Azure.WebPubSub.Common.csproj

```diff
@@ -13,9 +13,13 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="System.Memory.Data" />
```

Comment:
This file seems to set ``$(RequiredTargetFrameworks)`` to ``netstandard2.0`` should it add ``net6.0``?

Reply:
No;  `System.Memory.Data` is needed as a package.   The version referenced is from this repository and has some behavioral differences to what was brought forward into .NET.